amd_openvx_extensions/amd_rpp/include/internal_rpp.h

```diff
@@ -60,14 +60,14 @@ struct RPPCommonHandle {
 #elif ENABLE_HIP
```

Comment:
Can you please change this structure name to vxRppHandle instead? RPPCommonHandle and RppHandle is confusing

Reply:
Renamed the structure in the latest commit